### PR TITLE
📈 Use weekly oracle in CreditModel

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: test/utils/setupTruefi2.ts [hunk 1]
```diff
@@ -124,6 +128,10 @@ export const setupTruefi2 = async (owner: Wallet, provider: MockProvider, custom
   await mockSpotOracle.mock.getRate.withArgs(feeToken.address).returns(300)
   await weeklyFillBaseRateOracles(standardBaseRateOracle, feeBaseRateOracle, provider)
 
+  await mockTruOracle.mock.latestRoundData.returns(0, parseTRU(0.25), 0, 0, 0)
+  await timeTravel(provider, DAY * 8)
+  await weeklyTruPriceOracle.update()
```
Comment (skalermo) [inline test/utils/setupTruefi2.ts:133]: You could use here something like `weeklyFillBaseRateOracles`.

File: test/truefi2/lines-of-credit/CreditModel.test.ts [hunk 2]
```diff
@@ -387,7 +432,7 @@ describe('CreditModel', () => {
       ].map(([collateral, ltvRatio, result]) =>
         it(`when ${collateral} TRU is staked with ltvRatio=${ltvRatio}%, borrow limit rises by up to $${result}`, async () => {
           await creditModel.setStakingConfig(ltvRatio * 100, 0)
-          expect(await creditModel.conservativeCollateralValue(mockPool.address, parseTRU(collateral))).to.equal(parseEth(result))
+          expect(await creditModel.conservativeCollateralValue(parseTRU(collateral))).to.equal(parseEth(result))
```
Comment (MaciejNadolski98) [inline test/truefi2/lines-of-credit/CreditModel.test.ts:435]: Add tests to check how oracle price affects `conservativeCollateralValue` `conservativeCollateralRatio`, `poolBorrowMax` and `effectiveScore` (followup task?)

File: test/truefi2/lines-of-credit/TimeAveragedTruPriceOracle.test.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ describe('TimeAveragedTruPriceOracle', () => {
   const MAX_BUFFER_SIZE = 365 + 1
   const COOLDOWN_TIME = DAY
 
-  async function setPrice (price: number) {
+  async function setPrice (price: number | BigNumber) {
```
Comment (MaciejNadolski98) [inline test/truefi2/lines-of-credit/TimeAveragedTruPriceOracle.test.ts:30]: BigNumberish?

File: contracts/truefi2/CreditModel.sol [hunk 4]
```diff
@@ -298,7 +309,8 @@ contract CreditModel is ICreditModel, UpgradeableClaimable {
         if (borrowedAmount == 0) {
             return 0;
         }
-        return min(conservativeCollateralValue(pool, stakedAmount).mul(BASIS_POINTS).div(borrowedAmount), BASIS_POINTS);
+        uint256 borrowedInUsd = pool.oracle().tokenToUsd(borrowedAmount);
```
[the remark this answers is not in the document]
Comment (skalermo) [reply contracts/truefi2/CreditModel.sol:312]: Nevermind. Didn't notice it was *token* being converted to USD.

**Review by yuchenlintt — APPROVED** [review]

**Pull request merged**